cengage/react-magma
# chore: Update @emotion dependency

Issue: #1411 

## What I did

Moved @emotion from dependency to peerDependency in react-magma-dom

## Screenshots:


## Checklist 

## How to test

Once merged, we can try upgrading the version of an app that uses Magma to see if there are any errors.
